terminal/terminal_other.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
@derekparker I tried your suggestion, but it does not work:

```
$ git status
On branch fix_console_output
Your branch is up-to-date with 'origin/fix_console_output'.

Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        renamed:    terminal_other.go -> terminal_unix.go

$ git diff HEAD
diff --git a/terminal/terminal_other.go b/terminal/terminal_other.go
deleted file mode 100644
index 6287e06..0000000
--- a/terminal/terminal_other.go
+++ /dev/null
@@ -1,13 +0,0 @@
-// +build !windows
-
-package terminal
-
-import (
-       "os"
-       "strings"
-)
-
-// supportsEscapeCodes returns true if console handles escape codes.
-func supportsEscapeCodes() bool {
-       return strings.ToLower(os.Getenv("TERM")) != "dumb"
-}
diff --git a/terminal/terminal_unix.go b/terminal/terminal_unix.go
new file mode 100644
index 0000000..a78986f
--- /dev/null
+++ b/terminal/terminal_unix.go
@@ -0,0 +1,11 @@
+package terminal
+
+import (
+       "os"
+       "strings"
+)
+
+// supportsEscapeCodes returns true if console handles escape codes.
+func supportsEscapeCodes() bool {
+       return strings.ToLower(os.Getenv("TERM")) != "dumb"
+}
$ GOOS=windows go build -o /dev/null
# github.com/derekparker/delve/terminal
./terminal_windows.go:10: supportsEscapeCodes redeclared in this block
        previous declaration at ./terminal_unix.go:9
a@spot: terminal $ git reset --hard
HEAD is now at 3fdfa4e terminal: do not use escape codes on windows unless they are supported
$ git status
On branch fix_console_output
Your branch is up-to-date with 'origin/fix_console_output'.

nothing to commit, working directory clean
$ GOOS=windows go build -o /dev/null
$
```

I don't think *_unix.go filename has any special meaning, like _windows.go or _linux.go. I also looked for similar files in GOROOT:
```
$ cat $GOROOT/src/os/file_unix.go | grep build
// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
$
```
Where does it say that _unix.go means something?

Also there is plan9 version of Go. :-)

Ale